_blogposts/2023-04-15-improving-interop.mdx

```diff
@@ -0,0 +1,383 @@
+---
+author: rescript-team
+date: "2023-04-15"
+title: Better interop with customizable variants
+badge: roadmap
+description: |
+  A tour of new capabilities coming in ReScript v11
+---
+
+ReScript v11 is around the corner, and it comes packed with new features that will improve interop with JavaScript/TypeScript. We will shine some light on each significant feature in a series of blog posts.
+
+Interop with JavaScript/TypeScript is a top priority for ReScript. Recently we've made some changes to the runtime representation of variants that'll allow you to use variants for a large number of new interop scenarios, zero cost. This is important, because variants is _the_ feature of ReScript, enabling great data modeling, pattern matching and more.
```

Comment:
This feels way too long. Just Make a bullet list of the two changes we will introduce, and explain the impact in the example sections (that's what you already do anyways, so this feels redundant).

The quicker the reader comes to meaningful examples, the better.